src/main/app/windowManager.js

```diff
@@ -433,6 +433,12 @@ class WindowManager extends EventEmitter {
         window.reload()
       }
     })
+    ipcMain.on('invalidate-image-cache-by-id', id => {
```

Comment:
You can directly pass the window (`focusedWindow`) from the menu and send the request without adding `invalidateImageCache` method.